loky/process_executor.py

```diff
@@ -123,8 +123,13 @@
 # Number of bytes of memory usage allowed over the reference process size.
 _MAX_MEMORY_LEAK_SIZE = int(1e8)
 
+# Minimum time interval between two consecutive gc calls. Analogous to
+# _MEMORY_CHECK_DELAY, but when psutil is not installed
+_GC_CHECK_DELAY = 1
+
```

Comment:
Wouldn't it be better to only have 1 parameter instead of both `_GC_CHECK_DELAY` and `_MEMORY_CHECK_DELAY`?

Reply:
Smthg like `_MEMORY_LEAK_CHECK_DELAY`

Reply:
If we keep both, you should change `_GC_CHECK_DELAY` to `_GC_CALL_DELAY`.